models/icub_feet_fixed/icub_feet_fixed.sdf

```diff
@@ -1,14 +1,14 @@
 <?xml version='1.0'?>
-<sdf version='1.4'>
-  <model name="iCub">
+<sdf version='1.5'>
+  <model name="iCub_feet_fixed">
     <include>
-      <uri>model://icub</uri>
-      <pose>0.0 0 0.6 0 0 3.14</pose>
+      <uri>model://@ROBOT_NAME@</uri>
```

Comment:
For clarity, I would append a `.in` at the end of the name of any file that needs to be configured by CMake's `configure_file` . What do you think @nunoguedelha ?

Reply:
Fully agree. I will push an update right away.

Reply:
Regarding the home position...
Each model `icub_fixed`, `icub_feet_fixed`, `icub_standing` will be spawned with the respective home position (change in a next branch).
Actually `icub_standing` is supposed to be used for the Yoga and walking demos. I can replace it by 2 models: `icub_standing_yoga` (or just `icub_yoga`) and `icub_walking`.